apache/metron
# METRON-1646 Sensor Stubs should work when kerberized

The sensor-stubs do not work in a kerberized development environment.  This makes it difficult to test issues related to kerberization.  Enhancing sensor-stubs to work in a kerberized environment will make testing in the development environment under kerberization much simpler.

## Testing

1. Spin-up a development environment.
1. Ensure that alerts are visible in the Alerts UI for both Bro and YAF.
1. Kerberize the environment.
1. Restart the sensor-stubs service.
    ```
    service sensor-stubs restart
    ```
1. Ensure that new alerts are visible in the Alerts UI for both Bro and YAF.


## Pull Request Checklist
